pkg/pipeline/run.go

```diff
@@ -57,12 +57,16 @@ func Run(pc *config.Provider, rootDir string) { // nolint:gocyclo
 		controllerPkgList = append(controllerPkgList, filepath.Join(pc.ModulePath, p))
 	}
 	count := 0
-	for group, versions := range resourcesGroups {
+	for shortGroup, versions := range resourcesGroups {
+		group := pc.RootGroup
+		if shortGroup != "" {
```

Comment:
We can consider trimming before comparison and concatenation.

Reply:
As synced, we can treat this as developer error if there is space there.